pkg/codegen/python/gen.go

```diff
@@ -677,7 +685,7 @@ func genPackageMetadata(tool string, pkg *schema.Package, requires map[string]st
 	fmt.Fprintf(w, "setup(name='%s',\n", pyPack(pkg.Name))
 	fmt.Fprintf(w, "      version='${VERSION}',\n")
 	if pkg.Description != "" {
-		fmt.Fprintf(w, "      description=\"\"\"%s\"\"\",\n", pkg.Description)
+		fmt.Fprintf(w, "      description=\"%s\",\n", sanitizePackageDescription(pkg.Description))
```

Comment:
Are we sure these sanitized descriptions are going to be safe to embed inside quotes?  

Do we want instead this?

```suggestion
		fmt.Fprintf(w, "      description=%q,\n", sanitizePackageDescription(pkg.Description))
```

Reply:
Yep, that's better